### PR TITLE
Update royalty free license to 1.1

### PR DESCRIPTION
**Pull request by aurindam**

### CODE REVIEW

**Comment by tronical** [comment]
> You need to remove LicenseRef-Slint-Royalty-free-1.0 as it is not used anymore.

We link to it from the blog post and perhaps other places :(. Removing it breaks those links.

**Comment by ogoffart** [comment]
> We link to it from the blog post and perhaps other places :(. Removing it breaks those links.

I suggest we keep it for at least a few month.
In the mean time, we can also fix our links by linking to the new one

**Comment by ogoffart** [comment]
> We can copy the 1.0 version of the license to the website and link to it there

No, the point to keep it is only to get the old links working.

> Reuse will flag that unused license in our tree each time its run -- and its part of our CI setup. So we would need to disable that if we want to keep the license in the same place in our source tree.

Yes, can we disable that?  (how about working around it by having at least one dummy file using that license)

File: LICENSES/LicenseRef-Slint-Royalty-free-1.1.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Slint Royalty-free Desktop and Web Applications License, version 1.0.1
```
[the remark this answers is not in the document]
Comment (tronical) [reply LICENSES/LicenseRef-Slint-Royalty-free-1.1.md]: That's fixed.

**Review by tronical — APPROVED** [review]

**Pull request merged**